src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand for defining object methods to improve code readability.

Instead of:
subtract: function(num) {
  // your code
},

Use:
subtract(num) {
  // your code
},